src/components/TodoForm.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+
+class TodoForm extends Component {
+  constructor() {
+    super();
+    this.state = { inputVal: "" };
+  }
+
+  render() {
+    return (
+      <form>
+        <input
+          type="text"
+          placeholder="enter task"
+          value={this.state.inputVal}
+          onChange={(e) => this.setState({ inputVal: e.target.value })}
+        />
+        <button type="submit" onClick={() => this.props.addTodo(this.inputVal)}>
```

Comment:
incearca in loc sa adaugi event pe button sa adaugi even pe `<form>` si anume onSubmit.  Tot aici o sa ai nevoie si de `e.preventDefault()` cand o sa faci onSubmit

Reply:
Am adaugat, oricum nu apar taskurile pe pagina, atunci cand le introduc in input